modules/ROOT/pages/index.adoc

```diff
@@ -2,17 +2,18 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
+:linkattrs:
 :keywords: platform, arm, rest, soa, saas, api, proxy, design, develop, anypoint platform, studio, mule, devkit, studio, connectors, auth, exchange, api design, apikit, raml, application network, anypoint, arm, rest, soa, saas, api, proxy
 
-Use Anypoint Platform to build an application network, which is a structured network of applications, data, and devices connected by reusable APIs.
+Use Anypoint Platform to build a structured application network that connects applications, data, and devices with reusable APIs.
 
 == Explore Anypoint Exchange
 
 The APIs you build in Mulesoft to integrate applications and services are, by design, reusable and built with enterprise security in mind.
-You can discover these APIs, as well connectors (connections to external data sources), samples, and templates on 
-https://www.anypoint.mulesoft.com/exchange/[Exchange]. These assets include helpful links to reference guides and user guides. 
+You can discover these APIs, as well as connectors, samples, and templates on 
+https://www.anypoint.mulesoft.com/exchange/[Exchange^].
```

Comment:
Is that caret (`^`)intended?

Reply:
yes, added to all for URL links (not xref:, just when we send them to a separate site). It opens in a new window. I tested in local build.

Reply:
Yep, was just testing it myself. Opens it in a new tab. 👍